test/util.go

```diff
@@ -0,0 +1,319 @@
+//go:build conformance || e2e || examples
```

Comment:
Do we really need the tag here?

Reply:
Yep they are used for build tags for e2e/conformance/examples test. If we delete them, they'd all encounter build failure.

Reply:
And the reason for this refactor is to let all new integration test suite with new tag not to rely on the "e2e" tag where previously those init and setup lived in.